src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/migration-guide.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Migrate from the Prometheus OpenMetrics integration
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure OpenMetrics
+metaDescription: Migration guidelines to move from Prometheus OpenMetrics to New Relic Prometheus Agent.
+---
+
+If you're using our [Prometheus OpenMetrics integration](/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-update-or-uninstall-your-prometheus-openmetrics-integration), `nri-prometheus`, you can migrate to [New Relic's Prometheus agent](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent), `newrelic-prometheus-agent`, to achieve the same monitoring experience.
+
+Since the `newrelic-prometheus-agent` is actually running the `Prometheus server` in agent mode, the configuration file structure has changed to offer more flexibility and the possibility to use any scrape related feature of the Prometheus server, such as support of many [authorization methods](https://github.com/newrelic/newrelic-prometheus-configurator/blob/f46f1be393c6f8a5cddab66bf3d8528a993718fa/docs/TargetAuthorization.md), [relabel configuration setups](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/setup-prometheus-agent/#metric-label-transformations), or [scaling Prometheus instances](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/scale-prometheus-instances).
+
+We translate our simplified configuration file to a Prometheus server configuration file, to give you an easier configuration experience. We suggest that you read the documentation to be able to better follow this migration guide. See how to [set up New Relic's Prometheus agent](/docs/integrations/prometheus-integrations/install-configure-prometheus-agent/setup-prometheus-agent).
+
+To perform the migration, adjust the configuration section in your `values.yml` file with the values in this document.
+
+## Dashboards and alerts [#migration]
+
+The metrics are sent in the same way that they were sent using the Prometheus OpenMetrics integration, `nri-prometheus`, but you need to modify some metadata that has changed between Prometheus OpenMetrics integration, `nri-prometheus`, and the Prometheus agent, `newrelic-prometheus-agent`.
```

Comment:
I would say that the metadata changed and they should take that into account, not that they need to modify that.